index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <a class="logo" href="#">My Logo</a>
+        </div>
+        <nav>
+            <div>
+                <img src="./images/menu.png" alt="menu icon">
```

Comment:
- [ ] You did a great job adding a `hamburger` button inside a `nav` tag👍🏻 , but right now it is inside a div tag, please kindly consider adding it into a `<button>` tag because it is supposed to be a button: 
 ```
<button>
  <img src="./images/menu.png" alt="menu icon">
</button>
```

Reply:
corrected,
I changed tag <div> to tag <button>